redis/client.py

```diff
@@ -3281,6 +3287,22 @@ def zunionstore(self, dest, keys, aggregate=None):
         """
         return self._zaggregate('ZUNIONSTORE', dest, keys, aggregate)
 
+    def zmscore(self, key, members):
```

Comment:
```suggestion
    def zmscore(self, key, members=[]):
```

Reply:
Sorry for the late reply.

I am ok with a default value proposal (for compatible reason). But not sure if using a mutable type is fine. This contains risk that the default value might be modified.

So maybe I should change it with a default value None and modify the function body accordingly?